app/views/admin/organizations/_form.html.haml

```diff
@@ -1,118 +1,88 @@
-        = f.text_field :ascend_name
-
-    .control-group
-      = f.label :slug, "Slug", class: "control-label"
-      .controls
-        %p
-          = @organization.slug
-
-    .control-group
-      .control-label
-        = f.label :website
-      .controls
-        = f.text_field :website
-    - if @organization.id.present?
-      - emails = @organization.users.pluck(:email)
-      - emails << ENV['AUTO_ORG_MEMBER'] unless emails.any?
-      .control-group.chosen-select
-        .control-label
-          = f.label :embedable_user_email
-        .controls
-          = f.select :embedable_user_email, emails, prompt: "Select email", selected: @embedable_email
-        .controls.extra-field
-          .less-strong
-            - if @organization.memberships.count == 0
-              You can use #{ENV['AUTO_ORG_MEMBER']} if
-              %br
-              there are no members of the organization
-
-    .control-group
-      .control-label
-        %label
-          Kind
-      .controls.radio-controls
-        - Organization.kinds.each do |kind|
-          %label.radio
-            = f.radio_button :kind, kind
-            = kind.humanize
-  .bike-fields
-    .control-group.padded
-      .control-label
-        = f.label :show_on_map
-      .controls
-        = f.check_box :show_on_map
-
-    .control-group.padded
-      .control-label
-        = f.label :lock_show_on_map, 'Lock shown on map'
-      .controls
-        = f.check_box :lock_show_on_map, style: "margin: 4px .5em 0 0;"
-        %small.less-strong
-          Restrict org from updating show on map above (e.g. they're known bad actors)
-    .control-group.padded
-      .control-label
-        = f.label :api_access_approved
-      .controls
-        = f.check_box :api_access_approved
-        %small.less-strong
-          to be able to post updates via API, check this too
+    .row
+      .col-md-6
+        .form-group.mt-4
+          = f.label :website
+          = f.text_field :website, class: "form-control"
+        - if @organization.id.present?
+          - emails = @organization.users.pluck(:email)
+          - emails << ENV['AUTO_ORG_MEMBER'] unless emails.any?
+          .form-group
+            = f.label :embedable_user_email
+            = f.select :embedable_user_email, emails, {prompt: "Select email", selected: @embedable_email}, class: "form-control"
+            .less-strong
+              - if @organization.memberships.count == 0
+                You can use #{ENV['AUTO_ORG_MEMBER']} if
+                %br
+                there are no members of the organization
+      .col-md-6.mt-4
+        %fieldset{:disabled => "disabled"}
+          .form-group
+            = f.label :slug, "Slug:"
+            %p
+              = @organization.slug
+    .row
+      .col-md-6.pl-4
+        .form-group
+          %label
+            Kind
+          %ul.list-unstyled
+            - Organization.kinds.each do |kind|
+              %li
+                = f.radio_button :kind, kind
```

Comment:
![UNADJUSTEDNONRAW_mini_b](https://user-images.githubusercontent.com/20424916/56384547-4171f000-61e2-11e9-9362-d08daec0f9af.jpg)

So, this took me a lot longer than it should have because I'm a ding dong. Couldn't get those labels clickable, but I did it, and... without the list it looks like this. Do you want me to keep it like this or use a list? Or is there some other radio functionality that makes it a list without doing using li's?

Reply:
![Image 4-18-19 at 2 42 PM](https://user-images.githubusercontent.com/20424916/56386725-3621c300-61e8-11e9-8b8b-afef89796c21.jpg)